tools/internal/compare_versions.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Read input params
+read -r -p 'Enter connector name: ' connector
+read -r -p 'Enter first connector version: ' version_1
+read -r -p 'Enter second connector version: ' version_2
+
+use_state(){
+  failed=false
+for i in $(seq 1 3)
+do read -n 1 -r -s -p "Start sync with state (y/n)?" choice
+case "$choice" in
+  y|Y ) failed=false ; return 1 ;;
+  n|N ) failed=false ; return 0 ;;
+  * ) printf "\n";;
+esac
+failed=true
+done
+
+if "$failed"; then
+   echo "Process finished with exit code 1"
+   exit 1
+fi
+}
+
+use_state
+if [ "$?" -eq 1 ]; then
+    state_status="with"
+    state="/config_files/state.json"
+else
+    state_status="without"
+    state='" "'
+fi
+
+
+printf "\nPulling connectors images:\n"
+
+docker pull airbyte/"$connector":"$version_1"
+docker pull airbyte/"$connector":"$version_2"
```

Comment:
Is there ever a scenario where you would be testing this on a pre-release (#dev) version?  If so, docker-pull won't work.  I think `docker run` will pull the image if needed, so you might be able to skip these

Reply:
shifted pull to run stage